Display CLI (#117)
pi-top/pi-top-Python-SDK: docs/cli_tools.rst

    Time Remaining: 104
    Wattage: -41

pi-top display
=========================

> On pi-top devices with a screen, it allows to control different display settings.

Reviewer: ```suggestion
This command provides a way to control different display settings on pi-top devices with a built-in screen.
```

.. code-block:: bash

    pi-top display [-h] {brightness,backlight,timeout}

Where:

-h, --help
    Show a help message and exits

brightness
    Control display brightness

backlight
    Control display backlight

timeout
    Set the timeout before the screen blanks in seconds (0 to disable)

pi-top display brightness
~~~~~~~~~~~~~~~~~~~~~~~~~~

> Using `pi-top display brightness` without arguments will return the current brightness value.

Reviewer: Request or change the value of the display's brightness.

Note: this only works for the original pi-top, pi-topCEED and pi-top [3]. The pi-top [4] FHD Touch Display uses hardware buttons to control the brightness, and is not controllable via this SDK.

Also explain that pi-top [3] is 0-16, pi-top [1] and CEED is 0-10.

.. code-block:: bash

    pi-top display brightness [-h] [-v] [-i] [-d]
                                 [brightness_value]

Where:

-h, --help
    Show a help message and exits

-v, --verbose
    Increase verbosity of output

-i, --increment_brightness
    Increment screen brightness level

-d, --decrement_brightness
    Decrement screen brightness level

brightness_value
    Set screen brightness level [1-10] on pi-topHUB, or

>     [1-16] or pi-topHUB v2

Reviewer: Let's use end product names, not hub generations. Most people won't know what this means!

Example
~~~~~~~~~~~~~~~~~

.. code-block:: bash

    pi@pi-top:~ $ pi-top display brightness
    16

> pi-top display brightness

Reviewer: ```suggestion
pi-top display backlight
```

Reviewer: ```
pi-top display backlight on
pi-top display backlight 1
pi-top display backlight off
pi-top display backlight 0
```

~~~~~~~~~~~~~~~~~~~~~~~~~~

> 

Reviewer: Request or change the value of the display's brightness.

Note: this only works for the original pi-top, pi-topCEED and pi-top [3]. The pi-top [4] FHD Touch Display uses hardware buttons to control the brightness, and is not controllable via this SDK.

Reviewer: Also explain that pi-top [3] is 0-16, pi-top [1] and CEED is 0-10.

Using `pi-top display backlight` without arguments will return the current backlight status.

.. code-block:: bash

    pi-top display backlight [-h] [-v] [{0,1}]

Where:

-h, --help
    Show a help message and exits

-v, --verbose
    Increase verbosity of output

{0,1}
    Set the screen backlight state [0-1]

> pi-top display tiemout

Reviewer: ```suggestion
pi-top display timeout
```
Ctrl+F check for this

> pi-top display timeout

Reviewer: ```suggestion
pi-top display blank_time
```
?

Reviewer: ```
pi-top display backlight on
pi-top display backlight 1
pi-top display backlight off
pi-top display blank off
pi-top display blank off
```

> pt-brightness

Reviewer: I think that deprecated commands shouldn't even be discussed in the docs. We should put any information that people using those old commands would need into the print-out when the user attempts to use them (`.deprecated`).

Reviewer: We could print something like this:
```suggestion
To learn about the command arguments, check `pi-top display --help`.
```
